orion/pkg/router/handler.go

```diff
@@ -41,6 +42,7 @@ func (h *Handler) SetupApiEndpoints() {
 		apiLocations.GET("/v1/location/:locId", controllers.GetLocationById)
 		apiLocations.POST("/v1/location/:locId", controllers.UpdateLocation)
 		apiLocations.DELETE("/v1/location/:locId", controllers.DeleteLocation)
+		apiLocations.POST("/v1/publish", controllers.PublishLocations)
```

Comment:
This endpoint should be grouped with those with similar URLS and then sorted by GET, POST, etc. I put my publish endpoints after the GetALl.